pyspecProcScripts/lorentzian_to_gaussian.py

```diff
@@ -48,4 +48,5 @@ def L2G(s, axis='t2',
         fl.plot(temp.C.ft('t2'),
                 alpha=0.6,
                 label='after')
+        axhline(y=0,color='k')
```

Comment:
I was concerned that the baseline was depressed near the sides of the peak -- this might be a slight issue, but adding the axhline actually shows it's not quite as bad as I thought.